lib/px/utils/pxpayload.lua

```diff
@@ -196,6 +196,10 @@ end
 -- returns one value - table
 function PXPayload:decode(data)
     local fields = self.cjson.decode(data)
+    if fields == nil or next(fields) == nil then
```

Comment:
@wizzard Please check if its fine

Reply:
nope, that's not correct. 
please use the following structure: 
```
local ok, fields = pcall(self.cjson.decode,data)
if not ok then
```

Reply:
This is a really old PR. Should I close it? or can you add your code there, please?